lisa/schema.py

```diff
@@ -874,7 +874,7 @@ def check(self, capability: Any) -> search_space.ResultReason:
         if self.excluded_features:
             for feature in self.excluded_features:
                 cap_feature = self._find_feature_by_type(
-                    feature.type, capability.excluded_features
```

Comment:
Why change here?

Reply:
The logic seems wrong. We need to throw error if the unsupported feature in requirement is present in supported feature in capabilities.